keep-ui/app/settings/create-api-key-modal.tsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import { useForm, Controller, SubmitHandler, FieldValues } from 'react-hook-form';
+import { Dialog } from "@headlessui/react";
+import { TextInput, Button, Subtitle, SearchSelect, SearchSelectItem, Icon } from "@tremor/react";
+import { AuthenticationType } from "utils/authenticationType";
+import { User } from "./models";
+import { getApiURL } from "utils/apiUrl";
+import { InfoCircledIcon } from "@radix-ui/react-icons";
+
+const roleOptions: RoleOption[] = [
+  {
+    value: "cli",
+    label: "CLI",
+    tooltip: "CLI has write alert ability",
```

Comment:
CLI has admin ability for Keep management purposes (configure from API, etc)

Reply:
fix: 5d26d9669eb1b31915e792d0cf7ab263623f62b3